src/ucs/sys/sock.c

```diff
@@ -82,6 +83,22 @@ int ucs_netif_is_active(const char *if_name)
     return ucs_netif_flags_is_active(ifr.ifr_flags);
 }
 
+unsigned ucs_netif_bond_ad_num_ports(const char *bond_name)
+{
+    ucs_status_t status;
+    long ad_num_ports;
+
+    status = ucs_read_file_number(&ad_num_ports, 1,
```

Comment:
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/label=hpc-test-node-legacy,worker=0/14857//artifact/cov_build_devel_14857/output/errors/1/1sock.c.html#error

what about the following fix?
```
    ...
    ucs_assert_always(ad_num_ports <= UINT_MAX);
    return (unsigned)ad_num_ports;
}
```